Resources/config/schema/doctrine-1.0.xsd

```diff
@@ -47,6 +47,7 @@
         <xsd:attribute name="port" type="xsd:string" />
         <xsd:attribute name="user" type="xsd:string" />
         <xsd:attribute name="password" type="xsd:string" />
+        <xsd:attribute name="application_name" type="xsd:string" />
```

Comment:
the XML format always uses dashes in places where YAML uses underscores (it is normalized by the config system). What makes sense for users is to respect the XML convention (they will get autocompletion based on the XSD in their IDE anyway)